app-containers/docker/autobuild/postinst

```diff
@@ -6,6 +6,7 @@ if [[ "$1" = "install" ]]; then
 fi
 
 # Remove /etc/conf.d/docker 
+# Before this version (27.3.1+tini0.19.0-1) we added wrong config file in /etc, after this version we fixed it, don't change this version.
```

Comment:
Before 27.3.1+tini0.19.0-1, we shipped an systemd environment file (/etc/conf.d/docker) which rendered custom Docker configurations unusable. We removed this file in its next revision. Do not change this version.